origin-trials/downstream-trials.json

```diff
@@ -8,6 +8,14 @@
             "issue": "Acquisition Info",
             "flag": "msAcquisitionInfo"
         },
+        {
+            "label": "Ad Selection API",
+            "expiration": "2025-10-01",

```

Comment:
This date is further out than we'd normally do.
Usually an OT would run 6 milestones, e.g. the end date for 131-136 would be ~2025-07-15.